src/hooks/usePoolsStatuses.ts

```diff
@@ -56,8 +56,14 @@ export default function usePoolStatuses(): PoolStatuses {
         })
         .map((c) => c.paused())
       try {
-        const tvls = await ethcallProvider.all(supplyCalls)
-        const pausedStatuses = await ethcallProvider.all(pausedCalls)
+        const tvls = await ethcallProvider.tryEach(
+          supplyCalls,
+          supplyCalls.map(() => true),
```

Comment:
for safety / backwards compatibility's sake, lets mark each of these tryEach calls as `canFail=false`. As it currently is, one of these calls returning `null` will break stuff since our type doesn't account for possible failures. We should still get the performance boost from using the new contracts.